integrations/sparse/test_lucenesearcher_check_irst.py

```diff
@@ -36,19 +36,195 @@ def setUp(self):
         ibm_model_tar_name = 'ibm_model_1_bert_tok_20211117.tar.gz'
         os.system(f'wget {ibm_model_url} -P irst_test/')
         os.system(f'tar -xzvf irst_test/{ibm_model_tar_name} -C irst_test')
-        # qrel
-        self.qrels_path = f'{self.pyserini_root}/tools/topics-and-qrels/qrels.msmarco-passage.dev-subset.txt'
-
+        #wp term stat
+        wp_term_url = 'https://rgw.cs.uwaterloo.ca/JIMMYLIN-bucket0/data/bert_wp_term_freq.msmarco-passage.20220411.pickle'
+        os.system(f'wget {wp_term_url} -P irst_test/')
+        self.dl19_pass = 'tools/topics-and-qrels/topics.dl19-passage.txt'
+        self.dl20 = 'tools/topics-and-qrels/topics.dl20.txt'
+    
     def test_sum_aggregation(self):
```

Comment:
Can we separate dl19 and dl20 into separate functions (test cases). This way if there's an error, it'll be easier to track down and rerun.

Same for all cases below.